### PR TITLE
Update docs to use IDAES/examples instead of IDAES/examples-pse

### LINKED ISSUE #1131: Use IDAES/examples instead of IDAES/examples-pse
Anticipated areas to change:
- [ ] Documentation (where examples repo is, installation instructions)
      - PR: #1132
- [ ] `idaes get-examples` command: this either goes away entirely or, a friendlier solution, is made a simple wrapper on a `pip install idaes-examples` command followed by printing out where they got installed (i.e. output of `idaesx where`)

Probably one PR to handle both of the above.

### PR DESCRIPTION
**Pull request by dangunter**
Fixes #1131 

## Summary/Motivation:
Documentation needs to point to the new examples

## Changes proposed in this PR:
- Fix links and associated description
- Update installation steps

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Comment by ksbeattie** [comment]
Nitpick: I kinda liked the diataxis table with the somewhat smaller fonts...

**Comment by andrewlee94** [comment]
@dangunter It looks like the docs build for this PR has expired; could you fix the conflict so that the docs can be rebuilt. Note that our supported Python versions are now 3.8-3.11.

**Comment by dangunter** [comment]
@andrewlee94 @MAZamarripa and @lbianchi-lbl this is ready for review now

**Review by andrewlee94 — CHANGES_REQUESTED** [review]
Only one minor comment on updating the supported Python versions, but otherwise looks good to me.

File: docs/conf.py [hunk 1]
```diff
@@ -138,9 +138,10 @@
 # Used here to define substitutions for re-used URLs, e.g. just add "|examples-site|" to
 # any page and it will be replaced with the hyperlink and text shown below.
 rst_epilog = """
-.. |examples-site| replace:: `examples online documentation page`_
-.. _examples online documentation page: https://idaes.github.io/examples-pse/latest/index.html
-
+.. |python-min| replace:: 3.7
```
[the remark this answers is not in the document]
Comment (dangunter) [reply docs/conf.py:141]: there seems to be a second way to do all these replacements in the class RobustReplacements. I don't really understand why that's there (not impossible it's my doing, but I don't remember why if so). @lbianchi-lbl is this something you know more about?

[the remark this answers is not in the document]
Comment (lbianchi-lbl) [reply docs/conf.py:141]: Yeah, the simple (ie non-`Robust`) replacements are not expanded in code blocks, and we need the python version to be templated for `conda install python=3.X` commands.

[the remark this answers is not in the document]
Comment (dangunter) [reply docs/conf.py]: I updated it so they both use the same version constants

**Review by andrewlee94 — APPROVED** [review]

**Review by ksbeattie — APPROVED** [review]
LGTM

**Pull request merged**